metadata.json

```
@@ -21,6 +21,10 @@
       "name": "puppet/staging",
       "version_requirement": ">= 2.0.1 < 4.0.0"
     },
+    {
+      "name": "puppet/mysql_java_connector",
+      "version_requirement": ">= 3.0.2"
```

please also add an upper version constraint:
```suggestion
      "version_requirement": ">= 3.0.2 < 4.0.0"
```